gosrc/path.go

```diff
@@ -57,3 +57,9 @@ func IsGoRepoPath(path string) bool {
 func IsValidPath(importPath string) bool {
 	return pathFlags[importPath]&packagePath != 0 || IsValidRemotePath(importPath)
 }
+
+// IsStdlibPath returns true if path is in the standard library or $GOROOT
```

Comment:
Sorry, I had interpreted IsGoRepoPath as GOROOT path because of some priming I did while looking at some other code. Fixing.